packages/app/app.d.ts

```diff
@@ -1 +1,3 @@
 /// <reference types="nativewind/types" />
+
+declare module 'ethereumjs-util';
```

Comment:
not sure if this one is needed as well, the one that worked in lint was in `apps/next`